naxa-developers/tasking-manager
# facebook/chore/update rapid

- Restructured API Endpoints for ohsomeNow stats
- Implementing production API endpoints for ohsomeNow stats retrieval
- Simplify API request URL and update statistics base URL
- Add `StatsTimestamp` component
- Revise generic tooltip (ohsomeNow Stats) message
- Update test cases to reflect API restructuring
- Fix UTC to user location conversion issue in My Contributions
- feat: add tooltip to ohsomeNow stats section in homepage
- style ohsomeNow stat tooltip in home page
- add TimeZone information to project stats Tooltip
- refactor ohsomeNow stat tooltip to use same StatsTimestamp component
- fix stats tooltip typo
- Revert changes to yarn.lock in #6021 (#6076)
- replace zero with - for ohsomeNow stats no value case in My Contributions
- fix test case fail error
- Fix syntax error parsing JSON string
- Update CircleCI AWS CLI Orb syntax
- Remove Postgis extension creation
- fix: unable to access draft and private projects
- Enable IPv6 for Application Load Balancers
- Enable IPv6 for RDS databases
- Make RDS databases publicly inaccessible
- Remove IPv6 for databases
- Roadmap link updated
- Roadmap link update
- Fix My Tasks list view not showing pagination issue
- Do not show pagination when no data present in ProjectCardPaginator
- Fix test case failure issue for projectCardPaginator.test
- Fix crash issue when user logout from settings page
- Fix incomplete contributor list for validating multiple tasksin tasks comments issue
- Fix sonarlint error
- Fix username tags not displaying in comment section in project Q&A
- Address Sonarlint error in Questions and Comments section
- fix: inconsistent mapper level label on user profile page
- updating latest label names
- Override tachyons styles for code tag
- fix: associated projects in teams page
- fix: team delete message for associated projects
- fix: reformatted using black
- Remove references to rapid context methods that no longer exist.
- Rapid v2: Show the GPX data
- Fix NPE in rapidEditor
- Rapid updates
- Bump rapid version to 2.0.3, also make use of imagery and init promises to perform imagery switching at startup time. Also, rewrite history change to use URLSearchParams to replace existing Bing Param, if any.
- Fix bug that would cause the 'custom editor' useEffect code to fire even though no custom imagery source was defined.
- Update yarn.lock
- Rapid: Update for next Rapid version
- Update Rapid to v2.1.1
- Rapid: Use system interface for setting up Rapid
- Rapid: Load css and data resources from CDN
- Rapid: Fix issue where toggling sidebar would lose data, ensure mapview is resized
- Don't try to upload to sentry if we aren't logged in
- Rapid: Fix preauth
- Rapid: Use URL hash parameters instead of function calls
- Rapid: Get it working between task switches
- Rapid: Store dom node in state for reuse
- Rapid: Export functions that may be useful for iD and add basic tests for some of those functions
- Listen for reset events to update task state disable state
- rapid: Add PropTypes for SonarLint
- Update sidebar test for Rapid changes
